src/redis.go

```diff
@@ -61,23 +63,33 @@ func main() {
 		os.Exit(0)
 	}
 
-	redisURL := net.JoinHostPort(args.Hostname, strconv.Itoa(args.Port))
+	dialOptions := getStandardDialOptions(args.Password)
+	tlsDialOptions := getTLSDialOptions(args.UseTLS, args.TLSInsecureSkipVerify)
```

Comment:
It might be a bit confusing that `getTLSDialOptions` is called inconditionally. What do you think about simply checking
```
if args.UseTLS {
    options = append(options, redis.DialUseTLS(true))
}
```
directly in main? I think it might be more obvious when reading it.

Reply:
getTLSDialOptions already checks that, but it is true that it could lead to confusion, let me update that!